README.md

```diff
@@ -19,4 +19,9 @@ In the meantime, you can install and use this the Legacy OmniSharp extension to
     * Click the [X] to the right of the C# extension.
     * Allow Visual Studio Code to restart. 
     
-3. This extension has a dependency on Mono >= 4.0.1.
\ No newline at end of file
+3. This extension has a dependency on Mono >= 4.0.1.
+
+4. This extension can be used to target ASP.NET 5 RC1 and older DNX-based projects. If you need to install ASP.NET 5 RC1, you can still find it in the following locations:
```

Comment:
Don't you need ASP.NET 5 RC1 to run OmniSharp? Or will it somehow run on straight mono if it isn't installed?